js/admin.js

```diff
@@ -37,7 +40,10 @@
 						$( '.bfa-ajax-response-holder' ).html( response ).slideDown().delay(2000).fadeTo(600, 0).delay(300).slideUp().fadeTo(0, 100);
 					});
 				}
-			);
+			).fail( function(response ) {
+				alert( response.responseText );
+				location.reload();
+			});
```

Comment:
I'm not strictly opposed to this solution for now, however it seems like there's probably a better way we could reuse https://github.com/MickeyKay/better-font-awesome/pull/41/files?w=1#diff-badca728eb42ee201fabccc599273da9bde0c6162fa11eee73dbdcfd0e051413R40 to dynamically show a message instead of relying on the native `alert`, which feels kinda strange to me.

Any reason we can't simply re-use the preceding block:
```
$( '.bfa-loading-gif' ).fadeOut( function() {
	$( '.bfa-ajax-response-holder' ).html( response ).slideDown().delay(2000).fadeTo(600, 0).delay(300).slideUp().fadeTo(0, 100);
});
```
. . . and then simply mimic what we're doing in https://github.com/MickeyKay/better-font-awesome/pull/41/files?w=1#diff-0a856fc4f1eaca0d3148e3a297fbf6908416292ffc652af00883b697c140abb4R500 right before https://github.com/MickeyKay/better-font-awesome/pull/41/files?w=1#diff-0a856fc4f1eaca0d3148e3a297fbf6908416292ffc652af00883b697c140abb4R500 so that we echo/render the same style update/error message? In this case, we should be able to do basically the same thing I'd think, and maybe just update it to be red with the appropriate error message ala:
```
echo '<div class="error"><p>' . esc_html__( 'Settings were not saved due to a missing nonce.', 'better-font-awesome' ) . '</p></div>';
```

Reply:
That could work. The intention of having an alert (I hate them too) is that this error would occur only on super weird condition, it shouldn't occur. And if it does occur, we were forcing the user to read the alert, and then the browser would autorefresh to get a new nonce. Otherway the user would see the error, click on save again, and get the error again (unless the user manually refreshes the page).